code/game/objects/machinery/vending/vending_types.dm

```diff
@@ -381,32 +461,21 @@
 		/obj/item/tool/weldingtool = -1,
 		/obj/item/tool/wirecutters = -1,
 		/obj/item/tool/wrench = -1,
-		/obj/item/analyzer = -1,
-		/obj/item/t_scanner = -1,
-		/obj/item/tool/screwdriver = -1
+		/obj/item/tool/screwdriver = -1,
 	)
 
 /obj/machinery/vending/engivend
 	name = "Engi-Vend"
 	desc = "Spare electronics vending. What? Did you expect some witty description?"
 	icon_state = "engivend"
 	icon_deny = "engivend-deny"
-	req_access = list(ACCESS_MARINE_ENGINEERING)
-	products = list(/obj/item/clothing/glasses/meson = 2,/obj/item/multitool = 4,/obj/item/circuitboard/airlock = 10,/obj/item/circuitboard/apc = 10,/obj/item/circuitboard/airalarm = 10, /obj/item/circuitboard/general = 20, /obj/item/cell/high = 10)
-	contraband = list(/obj/item/cell/potato = 3, /obj/item/storage/belt/utility = 3)
-
-//This one's from bay12
-/obj/machinery/vending/engineering
-	name = "Tool Storage Machine"
-	desc = "A large storage machine containing various tools and devices for general repair."
-	icon_state = "engi"
-	icon_deny = "engi-deny"
-	req_access = list(ACCESS_MARINE_ENGINEERING)
-	products = list(/obj/item/clothing/head/hardhat = 4,/obj/item/storage/belt/utility = 4,/obj/item/clothing/glasses/meson = 4,/obj/item/clothing/gloves/yellow = 4,
-					/obj/item/tool/screwdriver = 12,/obj/item/tool/crowbar = 12,/obj/item/tool/wirecutters = 12,/obj/item/multitool = 12,/obj/item/tool/wrench = 12,
-					/obj/item/t_scanner = 12, /obj/item/cell = 8, /obj/item/tool/weldingtool = 8,/obj/item/clothing/head/welding = 8,
-					/obj/item/light_bulb/tube = 10, /obj/item/stock_parts/scanning_module = 5,/obj/item/stock_parts/micro_laser = 5,
-					/obj/item/stock_parts/matter_bin = 5,/obj/item/stock_parts/manipulator = 5,/obj/item/stock_parts/console_screen = 5)
+	products = list(
```

Comment:
mesons are still used, give belts as well. I don't remember there being belts in other vendors

Reply:
Is there a way to make it give out full belts? Right now it only gives out empty belts which seems somewhat annoying.

Reply:
Also I thought the stance was Mesons are bad? If I'm uninformed I'll readd them.

Reply:
I moved mesons into engi vend and a full belt into the clothing vendor